go.mod

```diff
@@ -1,32 +1,35 @@
 module github.com/openshift/ingress-node-firewall
```

Comment:
When I run the following commands:

```sh
% go version
go version go1.22.5 linux/amd64
% go mod tidy
% git diff
```

I see the following updates to my `go.mod` and `go.sum` files that are not reflected in the commit:

```diff
diff --git a/go.mod b/go.mod
index bbbe3f1b..88b97052 100644
--- a/go.mod
+++ b/go.mod
@@ -14,7 +14,6 @@ require (
        github.com/kennygrant/sanitize v1.2.4
        github.com/onsi/ginkgo v1.16.5
        github.com/onsi/gomega v1.33.0
-       github.com/openshift/api v0.0.0-20240605201059-cefcda60d938
        github.com/pkg/errors v0.9.1
        github.com/prometheus/client_golang v1.18.0
        github.com/prometheus/common v0.47.0
diff --git a/go.sum b/go.sum
index 5064402c..07249f16 100644
--- a/go.sum
+++ b/go.sum
@@ -128,8 +128,6 @@ github.com/onsi/gomega v1.7.1/go.mod h1:XdKZgCCFLUoM/7CFJVPcG8C1xQ1AJ0vpAezJrB7J
 github.com/onsi/gomega v1.10.1/go.mod h1:iN09h71vgCQne3DLsj+A5owkum+a2tYe+TOCB1ybHNo=
 github.com/onsi/gomega v1.33.0 h1:snPCflnZrpMsy94p4lXVEkHo12lmPnc3vY5XBbreexE=
 github.com/onsi/gomega v1.33.0/go.mod h1:+925n5YtiFsLzzafLUHzVMBpvvRAzrydIBiSIxjX3wY=
-github.com/openshift/api v0.0.0-20240605201059-cefcda60d938 h1:3Y0E8q6pFb8PgzW0N52jVZBollMeW1kI13iU/vXPCnc=
-github.com/openshift/api v0.0.0-20240605201059-cefcda60d938/go.mod h1:OOh6Qopf21pSzqNVCB5gomomBXb8o5sGKZxG2KNpaXM=
 github.com/pkg/errors v0.9.1 h1:FEBLx1zS214owpjy7qsBeixbURkuhQAwrK5UwLGTwt4=
 github.com/pkg/errors v0.9.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINEl0=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
```

Do you need to run `go mod tidy` and `go mod vendor` again to ensure these updates are correctly reflected in the commit?